docs/tutorials/python_userguide/coherence.txt

```diff
@@ -1,2 +1,71 @@
-6. Tokens Coocurancy and Coherence Coumputation
-=======
+6. Tokens Coocurancy and Coherence Computation
+==============================================
+
+* **Interpretability and Coherence of Topics**
+
+The one of main requirements of topic models is interpretability (i.e., do the topics contain tokens that, according to subjective human judgment, are representative of single coherent concept). `Newman at al. <http://www.aclweb.org/anthology/N10-1012>`_ showed the human evaluation of interpretability is well correlated with next automated quality measure called coherence. The coherence formula for topic is defined as
```

Comment:
Is it better to say *... is well correlated with **the following** automated quality measure ...* ?